lua/caramba/orchestrator.lua

```
@@ -0,0 +1,278 @@
+	for _, t in ipairs(delta.current_tasks or {}) do
+		if type(t) == 'string' then add_unique(plan.current_tasks, { description = t }) else add_unique(plan.current_tasks, t) end
+	end
+	for _, k in ipairs(delta.known_issues or {}) do add_unique(plan.known_issues, k) end
+	-- persist
+	planner.save_project_plan()
+end
+
+-- Ask LLM for plan deltas
+local function request_plan_delta(prompt_text, callback)
+	local messages = {
+		{ role = 'system', content = [[You maintain a concise implementation plan. Given input, output ONLY JSON with keys: goals[], current_tasks[], known_issues[]. No prose.]] },
```

Luacheck (1.2.0)

[warning] line is too long (174 > 120)

(W631)